packages/block-library/src/media-text/edit.js

```diff
@@ -318,6 +365,17 @@ function MediaTextEdit( { attributes, isSelected, setAttributes } ) {
 
 	const blockEditingMode = useBlockEditingMode();
 
+	function resetAll() {
+		setAttributes( {
+			isStackedOnMobile: false,
+			imageFill: false,
+			mediaAlt: undefined,
```

Comment:
This should be updated with an empty string (`''`). Otherwise, I don't think the alt text will be removed when you click "Reset all".